as/cNFT/assembly/market.ts

```diff
@@ -50,6 +54,14 @@ export function set_bid(tokenId: string, bid: Bid): Bid {
     assert_eq_attached_deposit(bid.amount)
     assert_token_exists(tokenId)
 
+    let contract_extra = storage.getSome<NFTContractExtra>(PersistentNFTContractMetadata.STORAGE_KEY_EXTRA)
+    assert(
+        u128.ge(bid.amount, u128.from(contract_extra.min_bid_amount)),
```

Comment:
This is not an issue cause on line _54_ we are checking that `bid.amount` equals `attached_deposit` but it would be better to use `context.attached_deposit` for this assert, instead of `bid.amount`